Neos.Flow/Classes/Mvc/View/JsonView.php

```diff
@@ -292,6 +293,8 @@ protected function transformObject($object, array $configuration)
                     $propertiesToRender[$propertyName] = $propertyValue;
                 } elseif (isset($configuration['_descend']) && array_key_exists($propertyName, $configuration['_descend'])) {
                     $propertiesToRender[$propertyName] = $this->transformValue($propertyValue, $configuration['_descend'][$propertyName]);
+                } elseif ($propertyValue instanceof \DateTimeInterface) {
+                    $propertiesToRender[$propertyName] = $propertyValue->format(\DateTime::ATOM);
```

Comment:
ISO8601 is not compatible with ISO-8601 as you might know (https://www.php.net/manual/en/class.datetimeinterface.php). I am currently working on a separate commit to make this breaking change to ATOM for the formatting above.

Reply:
Yep. We should probably just keep this format change separate from this. If we merge this as is, it would change the format of datetime properties that were previously output as (the wrong) ISO8601, since this else condition now happens before the "normal path".

Reply:
@albe What do you think about an object variable `$dateFormat`, to specify the format for date conversions. This would set to ISO8601, marked as deprecated for 8.0 and it could easily be set / overwritten in an own view implementation.

I read through the examples but I still don't get the `_descend` part. I pass the object directly to `value` and the object itself contains the date property. How should that configuration look like?

Reply:
> I read through the examples but I still don't get the _descend part.

You're not to blame. The coniguration is super weird. IIRC for this the configuration would need to look something like that:
```
[ 'value' => ['_descend' => [ 'dateTime' => [] ] ]
```
Yes, the datetime property needs to have an (empty) array as config at least.

> What do you think about an object variable $dateFormat, to specify the format for date conversions. This would set to ISO8601, marked as deprecated for 8.0 and it could easily be set / overwritten in an own view implementation.

Sounds like a good plan to correct at least this date format issue in a b/c way 👍

Reply:
Addendum: Let's maybe even make it a configuration option of this view like `'jsonEncodingOptions'` - that way no own view needs to be implemented/inherited